modules/simple_forms_api/spec/support/shared_examples_for_base_form.rb

```diff
@@ -13,7 +13,7 @@
     end
 
     context 'address is present and not in US' do
-      let(:data) { { address_key => { 'country' => 'Canada' } } }
+      let(:data) { { address_key => { 'country' => 'CAN' } } }
```

Comment:
This was not representative of the data we were receiving from form submissions.